apps/settings/src/views/Users.vue

```diff
@@ -445,51 +445,40 @@ export default {
 		async createGroup(gid) {
 			// group is not valid
 			if (gid.trim() === '') {
-				Vue.nextTick(() => {
-					this.toggleAddGroupEntry(true)
-				})
 				return
 			}
 
 			try {
 				this.loadingAddGroup = true
 				await this.$store.dispatch('addGroup', gid.trim())
 
-				this.toggleAddGroupEntry(false)
-				this.$router.push({
+				this.hideAddGroupForm()
+				await this.$router.push({
 					name: 'group',
 					params: {
 						selectedGroup: encodeURIComponent(gid.trim()),
 					},
 				})
 			} catch {
-				this.toggleAddGroupEntry(true)
+				this.showAddGroupForm()
 			} finally {
 				this.loadingAddGroup = false
 			}
 		},
 
-		/**
-		 * Toggle the add group entry editing state
-		 * @param {boolean} [state] set state instead of toggling
-		 */
-		toggleAddGroupEntry(state) {
-			if (state === undefined) {
-				state = !this.$refs.addGroup.editing
-			}
-			this.$refs.addGroup.editing = state
-
-			// focus input
-			Vue.nextTick(() => {
-				if (this.$refs.addGroup.$el) {
-					const input = this.$refs.addGroup.$el.querySelector('form > input[type="text"]')
-					if (input) {
-						input.focus()
-					}
-				}
+		showAddGroupForm() {
+			this.$refs.addGroup.editingActive = true
+			this.$refs.addGroup.onMenuToggle(false)
+			this.$nextTick(() => {
+				this.$refs.addGroup.$refs.editingInput.focusInput()
 			})
 		},
 
+		hideAddGroupForm() {
+			this.$refs.addGroup.editingActive = false
+			this.$refs.addGroup.editingValue = ''
```

Comment:
```suggestion
			this.$refs.addGroup?.editingActive = true
			this.$refs.addGroup?.onMenuToggle(false)
			this.$nextTick(() => {
				this.$refs.addGroup?.$refs?.editingInput?.focusInput()
			})
		},

		hideAddGroupForm() {
			this.$refs.addGroup?.editingActive = false
			this.$refs.addGroup?.editingValue = ''
```
Some optional chaining safety :)

Reply:
Hmm. Can $refs.addGroup be null? I would argue that something is broken if addGroup is null ;)

Reply:
good point 🤔 hadn't looked over all of it yet, refs here won't be null when the function is called :)

Reply:
$ref is null when triggered before mounted or mounted finished, don't recall atm.